src/index.ts

```diff
@@ -209,11 +224,20 @@ export async function fixDuplicates(root: string) {
 }
 
 export async function listDuplicates(root: string) {
+  const config = await readConfig(root);
+  const ignoredSet = new Set(config?.ignored ?? []);
+  const isIgnored = (duplicate: Package) => ignoredSet.has(duplicate.name);
```

Comment:
There's a bit of duplicate code between `listDuplicates` and `fixDuplicates`. Is that something we'd want to factor out?